node/node.go

```diff
@@ -679,6 +679,10 @@ func (n *Node) GetAllESDTTokens(address string, options api.AccountQueryOptions,
 			continue
 		}
 
+		if esdtToken.Value.Cmp(big.NewInt(0)) <= 0 {
```

Comment:
:+1: 

However, maybe have a single (reusable) instance for the big int zero (just an idea, maybe it does not worth it)?

Reply:
How about using `Sign()`?

https://pkg.go.dev/math/big#Int.Sign

Reply:
Good idea, changed.